packages/data-models/appConfig.ts

```diff
@@ -106,8 +106,16 @@ const APP_ROUTE_DEFAULTS = {
   ],
 };
 
+type IHeaderColourOptions = "primary" | "secondary" | "none";
+
 const APP_HEADER_DEFAULTS = {
   title: "App",
+  collapsing: false,
```

Comment:
nit(non-blocking) probably more consistent to use `collapse` (imperative) instead of `collapsing` (present participle)